lib/src/base-styles/_nx-btn-bar.scss

```diff
@@ -8,6 +8,8 @@
 .nx-btn-bar {
   @include container-horizontal;
 
+  align-items: first baseline;
```

Comment:
I'd prefer `center` since it's clearer in its effect/intent

Reply:
> I'd prefer center since it's clearer in its effect/intent

8d20874